Civi/Api4/Action/GetActions.php

```diff
@@ -43,7 +43,7 @@ class GetActions extends AbstractAction {
 
   public function _run(Result $result) {
     $includePaths = array_unique(explode(PATH_SEPARATOR, get_include_path()));
-    $entityReflection = new \ReflectionClass('\Civi\API\V4\Entity\\' . $this->getEntity());
+    $entityReflection = new \ReflectionClass('\Civi\Api4\Entity\\' . $this->getEntity());
     // First search entity-specific actions (including those provided by extensions
     foreach ($includePaths as $path) {
       $dir = \CRM_Utils_File::addTrailingSlash($path) . 'Civi/Action/' . $this->getEntity();
```

Comment:
@mickadoo I think this also needs fixing. And the other one a couple lines below this.